src/RawRabbit/Configuration/Publish/PublishConfiguration.cs

```diff
@@ -9,5 +10,7 @@ public class PublishConfiguration
 		public ExchangeConfiguration Exchange { get; set; }
 		public string RoutingKey { get; set; }
 		public Action<IBasicProperties> PropertyModifier { get; set; }
+		public bool Mandatory { get; set; }
```

Comment:
We could derive the fact that the publish is mandatory if the `BasicReturn` event handler is not null. That way, we don't need to introduce the Mandatory property. Thoughts?